avm/res/storage/storage-account/main.json

```diff
@@ -4963,6 +5063,16 @@
       },
       "value": "[reference('storageAccount').primaryEndpoints]"
     },
+    "privateEndpointsNetworkInterfaceIds": {
```

Comment:
Both this file and the readme are currently not up to date with the `main.bicep`

Reply:
I executed Set-AVMModule to fix this